### ISSUE TITLE
Angular CLI incompatibility

### ISSUE THREAD

**Issue by RDUser-dev** [Siedlerchr/types-ol-ext#164]
Hi @Siedlerchr 
thanks for the grate job,
We have some problem with this package and ``the ng update`` command.
We have filled an issue to the Angular CLI team https://github.com/angular/angular-cli/issues/28373
but it was closed as not planned.
Basically Angular ng update not resolving the correct NPM package because of the NPM alias that your repo use for installation.
Do you have any possible suggestion for a workaround?

**Comment by javier-lopez-1s** [comment]
Have you tried uninstalling, updating, and then reinstalling again?
```
npm uninstall @types/ol-ext
ng update
npm i -D @types/ol-ext@npm:@siedlerchr/types-ol-ext
```

**Comment by RDUser-dev** [comment]
> Have you tried uninstalling, updating, and then reinstalling again?
> 
> ```
> npm uninstall @types/ol-ext
> ng update
> npm i -D @types/ol-ext@npm:@siedlerchr/types-ol-ext
> ```

yes it is the workaroud that we currently use to update angular.. but it is quite strange and cumbersome as a thing.
I think it would be enough to provide your types via https://github.com/DefinitelyTyped/DefinitelyTyped
but I don't know in detail how to do it

**Comment by MatteoBarbieriSacmi** [comment]
I'm experiencing the same problem with ng update. Any news?

**Comment by Siedlerchr** [comment]
I am sorry, the https://www.npmjs.com/package/types-ol-ext does not belong to me, it was registered by someone else, I have no idea

**Comment by RDUser-dev** [comment]
You're right @Siedlerchr, looking with `npm owner ls types-ol-ext` I actually see that the bundle belongs to a certain Anthony Bourque.
It would be nice to try to contact him to try to get the ownership back.
Anyway thanks for the work, but we were forced to remove the dependency on types-ol-ext from our project because of the issue.
In case in the future we will gladly reinsert it.
Thanks again
